### ISSUE TITLE
2.1.24 exporting receives "requested JSON parse failed"

### ISSUE THREAD

**Issue by bcwhittle** [nilsteampassnet/TeamPass#1168]
I am currently running 2.1.24. When I try to export passwords, it seems to work; the counter goes up to 100, but then it pops up a separate window with an error: "Requested JSON parse failed." This happens with both Chrome and Explorer. I am using an account with full access to all passwords. I have tried doing several different single folders as well as multiple folders.

When I look on the server, nothing has been added to the \files folder.

**Comment by florck** [comment]
Hello,
I have same problem. I can not find any error, neither on server or on the firebug console.
Can you please suggest what we should do?
I'm in the official docker, just changed the php5/apache2/php.ini to allow showing errors instead of 500 error. Thanks

**Comment by michmich9337** [comment]
Up,

I got the same problem in 2.1.25, is their a way to fix it ?

**Comment by henrybcp** [comment]
Anyone found a fix for this yet? I'm using 2.1.26 and I'm getting it as well.

**Comment by bcwhittle** [comment]
I finally found the problem in my installation. After looking at a possibly related issue (https://github.com/nilsteampassnet/TeamPass/issues/1128), I used the debug feature in Chrome and saw that where it was trying to save the .pdf looked wrong. When I checked my settings in TeamPass, I found that the directory for files was pointing to a directory that did not exist. After I changed that setting, I could successfully run an export.

Thanks, Nils, for your continued work on a great little password program!
Brian
